metaflow/plugins/conda/conda.py

```diff
@@ -182,9 +182,10 @@ def _call_conda(self, args, architecture=None, disable_safety_checks=False):
                 raise CondaException(err)
             except (TypeError, ValueError) as ve:
                 pass
+            print("***", e.output)
```

Comment:
Maybe, we need to leave this line out of the PR?

Reply:
uh sorry